### PR TITLE
actor: need to keep signature of `clearActorFields` for now

### PR DESCRIPTION
**Pull request by jrudolph**
Follow up to #27614 which disrupted the signatures.

### CODE REVIEW

**Comment by jrudolph** [comment]
I reinstalled the old arguments even if they are not used now any more.

**Review by raboof — APPROVED** [review]

File: akka-actor/src/main/scala/akka/actor/ActorCell.scala [hunk 1]
```diff
@@ -686,7 +686,9 @@ private[akka] class ActorCell(
     case _                               =>
   }
 
-  final protected def clearActorFields(): Unit = {
+  @InternalStableApi
+  @silent("never used")
+  final protected def clearActorFields(actorInstance: Actor, recreate: Boolean): Unit = {
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/actor/ActorCell.scala:691]: any mima filter that should be removed now?

**Review by patriknw — APPROVED** [review]
LGTM, with a question...

File: akka-actor/src/main/scala/akka/actor/ActorCell.scala (hunk 1 above)
Comment (jrudolph) [reply akka-actor/src/main/scala/akka/actor/ActorCell.scala:691]: Good point. Removed the line which is now obsolete.

**Pull request merged**